### DISCUSSION TITLE
Brpc error

### DISCUSSION THREAD

**Discussion by eleven-f** [apache/doris#43008] [A - General / Q&A]
What may cause the following error？
Doris version: 
    BE: doris-2.1.5-rc02-d5a02e095d
Error:
    1105, 'errCode = 2, detailMessage = (192.168.2.105)[CANCELLED]failed to send brpc when exchange, error=Resource temporarily unavailable, error_text=[E11]Resource temporarily unavailable @192.168.2.101:8060, client: 192.168.2.105, latency = 843'

**Comment by ixzc** [comment]
we have fixed this issue, https://github.com/apache/doris/pull/37888, please upgrade your doris to 2.1.6

**Comment by eleven-f** [comment]
version: 2.1.7 The error still exists。
![image](https://github.com/user-attachments/assets/1791dd40-ef2a-4081-9d7d-f0900c541538)
